examples/nemo/eg5/extract_kernels.py

```diff
@@ -91,12 +92,17 @@ def trans(psy):
             print(f"Invoke {invoke.name} has no Schedule! Skipping...")
             continue
 
-        count = 0
         for kern in sched.children:
-            count += 1
             if not isinstance(kern, NemoLoop):
                 continue
             try:
+                # The example contains array expressions, e.g.:
+                # zwx(:,:,jpk) = 0.e0
+                # PSyclone will convert them using the shape of zwx. But
```

Comment:
"PSyclone represents this internally using `Range` with `LBOUND` and `UBOUND` intrinsics and currently this results in several..."

Reply:
Done.